packages/batch-submitter/test/helpers/constants.ts

```diff
@@ -6,7 +6,7 @@ export const FORCE_INCLUSION_PERIOD_SECONDS = 600
 export const DEFAULT_ACCOUNTS = defaultAccounts
 export const DEFAULT_ACCOUNTS_HARDHAT = defaultAccounts.map((account) => {
   return {
-    balance: ethers.BigNumber.from(account.balance).toHexString(),
+    balance: account.balance,
```

Comment:
Are these equivalent?

Reply:
Sorry should have added this line in the PR description but essentially there was a breaking change in `hardhat` _while_ this PR was happening that meant the contracts unit test run broke.